data/api.py

```diff
@@ -215,6 +217,17 @@ class JobAnswerSetViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return JobAnswerSet.objects.filter(user=self.request.user)
 
+    @detail_route(methods=['post'])
```

Comment:
We should add specify the `JobSerializer` class in the detail_route call, so that the renderer knows to wrap this in a `job` rather than a `job-answer-set`